pkgs/applications/networking/cluster/multus-cni/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "multus-cni";
+  version = "3.7.1";
+
+  src = fetchFromGitHub {
+    owner = "k8snetworkplumbingwg";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "04rn7ypd0cw2c33wqb9wqy1dp6ajvcp7rcv7zybffb1d40mdlds1";
+  };
+
+  buildFlagsArray = let
+    multus = "gopkg.in/intel/multus-cni.v3/pkg/multus";
+    commit = "f6298a3a294a79f9fbda0b8f175e521799d5f8d7";
+  in [
+    "-ldflags=-s -w -X '${multus}.version=v${version}' -X '${multus}.commit=${commit}'"
+  ];
+
+  preInstall = ''
+      mv $GOPATH/bin/cmd $GOPATH/bin/multus
+  '';
+
+  vendorSha256 = null;
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
The test suite from multus-cni assumes an existing k8s cluster ready at local. This assumption does not always hold  especially when we can only build the package, such as using nixops in a non-nixos linux distro.

BTW, I've been verifying multus-cni (although the master rev) in a real use case for 1+ years in a project.
https://github.com/onixie/nikops/blob/master/src/addons/multus-cni.nix

Reply:
Please add a comment that the tests require a k8s cluster.

Reply:
Sure, comment added.